tests/Feature/RoomTest.php

```diff
@@ -17,6 +18,7 @@ class RoomTest extends TestCase
     public function test_getAllRooms_success(): void
     {
         Room::factory()->create();
+        Type::factory()->create();
```

Comment:
You probably don't need this anymore since your room factory creates it's own type automatically